imapsync.rst

```diff
@@ -44,8 +45,3 @@ Known limitations
 
 - Imapsync does not integrate with :ref:`Piler <piler-section>`: messages
   copied by Imapsync are not archived.
-
-- Sieve filters do not work with Imapsync, as they are evaluated during
-  LMTP delivery. Instead, Imapsync uses IMAP commands to interact directly
-  with Dovecot, bypassing LMTP and preventing the execution of Sieve
-  scripts.
```

Comment:
I'd add an explanation for it, something like:
```
Sieve filters are evaluated at the end, only on the inbox folder. The only limitation is that the "sieve_before" filter, which is used to move the email to spam, is not executed. This is intentional because spam control should be performed by the original mail server.
```

Reply:
I'd rather say that sieve filters are working, not matter the form you prefer: the user must be informed.